main.go

```diff
@@ -218,7 +219,16 @@ func main() {
 		os.Exit(1)
 	}
 
-	if err = controllers.NewImageStreamReconciler(client, filter, kernelOsDtkMapping).SetupWithManager(mgr); err != nil {
+	const dtkImageStreamNamespace = "openshift"
```

Comment:
WDYT about moving namespace and name into constants.go, and make ImageReconciler create the nsn?

Reply:
I think that would only make sense if we renamed the reconciler into something like `DTKImageStreamReconciler`. WDYT?

Reply:
since this is what it does anyway ( i don't think that it's code can work in case it is passed a different image stream), then let's rename it in a different PR